core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.html

```diff
@@ -90,8 +91,8 @@
             <td>
                 {{ rowData['modUserName'] }}
             </td>
-            <td>
-                {{ rowData['modDate'] }}
+            <td *dotRelativeDate="rowData['modDate']; data as result">
+                {{ result }}
```

Comment:
I think this shouldn't be a structural directive... this can easily be just a pure pipe we can use like this:

```suggestion
            <td>
                {{ rowData['modDate'] | relativeDate }}
```

And please update this pipe to be use in other places where we have relative dates, like the data-listing-datatable component.

Reply:
done!